src/implementing_new_features.md

```diff
@@ -111,7 +109,6 @@ for stabilization in a checklist, e.g.,
 - [ ] Stabilize the feature. ([See instructions on rustc-dev-guide.](https://rustc-dev-guide.rust-lang.org/stabilization_guide.html#stabilization-pr))
 ```
 
-<a name="stability-in-code"></a>
```

Comment:
Removing this shouldn't break anything because I believe the ID already matches the ID generated for the heading.